lumos/optimal_control/fixed_mesh_ocp.py

```diff
@@ -40,10 +40,34 @@ def __init__(
         # NOTE: we choose to make mesh_scale a variable instead of a config to make the
         # distinction of fixed_grid clearer!
         assert mesh_scale > 0.0, "mesh_scale must be a positive scalar!"
-        self._mesh_scale = mesh_scale
+        self.set_mesh_scale(mesh_scale)
 
         super().__init__(model=model, sim_config=sim_config)
 
+    def set_mesh_scale(self, mesh_scale: float):
+        """When we set the mesh_scale, some other properties must change"""
+        self._mesh_scale = mesh_scale
```

Comment:
test how much runtime increase we get from reverting to BaseConstraints vs LinearConstraints for continuity -> the cost of using LinearConstraints on our code structure is pretty big now

Reply:
without caching:
```sh
INFO:lumos.optimal_control.nlp:continuity.constraints: 0.001479
INFO:lumos.optimal_control.nlp:continuity.jacobian: 0.001209
INFO:lumos.optimal_control.nlp:continuity.hessian: 0.000003
```

with caching
```sh
INFO:lumos.optimal_control.nlp:continuity.constraints: 0.001271
INFO:lumos.optimal_control.nlp:continuity.jacobian: 0.001015
INFO:lumos.optimal_control.nlp:continuity.hessian: 0.000002
```

< 0.5ms per call difference, so negligible